presto-main/src/main/java/io/prestosql/operator/scalar/JoniRegexpFunctions.java

```diff
@@ -333,8 +333,8 @@ public static long regexpPosition(@SqlType("varchar(x)") Slice source, @SqlType(
     @LiteralParameters("x")
     @SqlType(StandardTypes.INTEGER)
     public static long regexpPosition(@SqlType("varchar(x)") Slice source,
```

Comment:
First argument should go on the next line

Reply:
This is not something the code formatter changes. There are many instances of this to fix across the code base and would need to be done separately (not sure how to automate it). But I can fix these by hand as part of this PR if you want.